### PR TITLE
adding bidoof event article

### PR DESCRIPTION
**Pull request by uday-kalyan-s**
adding the bidoof event article.

- [x] made entry in py_events
- [x] added route
- [x] made event page\
- [x] added needed assets

### CODE REVIEW

**Comment by wildcraft958** [comment]
Everything seems fine

File: templates/event-articles/bidoof-art.njk [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+{% set pagetitle = 'Bidoof Day Art Competition' %}
+{% set pagedesc = 'The Intra-Society Anime Quiz was an EPIC showdown, and the results? Mind-blowing!\n From jaw-dropping trivia to intense battles of otaku knowledge, we witnessed friendships forged and rivalries born!' %}
+
+{% block eventContent %}
+	<div class="event-info">
+		<span class="heading">Date:</span> June 20, 2024 <br>
+		<span class="heading">Location:</span> Online <br>
+		<span class="heading">Author:</span> Markio <br>
+	</div>
+	<h2> declaration </h2>
```
Comment (ItsAnkan) [inline templates/event-articles/bidoof-art.njk:12]: Title Case

File: templates/event-articles/bidoof-art.njk [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	<p>
+		During the summer vacation of 2024, MASK conducted its first online art competition;
+        you can probably guess the theme. Yes, it was <b>Bidoof</b>!
+        In celebration of Bidoof Day, which was on 1st July, we were organizing an art competition celebrating Bidoof.
+        <br>
+        The competition was divided into two sections, Traditional Art and Digital Art, and registration started on the 20th of May.
+        The competition Deadline was 20th June, and by the end, we had received many submissions.
+        After a long and rigorous grading process, the results were finally declared on 1st July, Bidoof Day.
+	</p>
+	<h2> results </h2>
```
Comment (ItsAnkan) [inline templates/event-articles/bidoof-art.njk:22]: "Results"

File: assets/events/bidoof-art.jpg
Comment (ItsAnkan) [inline assets/events/bidoof-art.jpg:1]: its too large, try to make it a bit smaller by compressing it

**Review by ItsAnkan — CHANGES_REQUESTED** [review]

**Review by destryptor — APPROVED** [review]

**Review by ItsAnkan — APPROVED** [review]
approved

**Comment by destryptor** [comment]
Merging, blame onkon if anything breaks (free trit automatically)

**Pull request merged**